hg-fast-export.py

```diff
@@ -171,7 +178,10 @@ def refresh_gitmodules(ctx):
   gitmodules=""
   # Create the .gitmodules file and all submodules
   for name,subrepo_info in ctx.substate.items():
-    gitmodules+=refresh_hg_submodule(name,subrepo_info)
+    if subrepo_info[2]=='git':
+      gitmodules+=refresh_git_submodule(name,subrepo_info)
+    elif submodule_mappings and name in submodule_mappings:
```

Comment:
The `and name in submodule_mappings` should go into it's own patch as it does more than preserving the current functionality. As far as I can tell it is a new optimization which became simple to implement following the previous refactoring.

Reply:
Can you comment on the previous review comment as you didn't make a change, am I wrong or did you just miss to adress it?

Reply:
My apologies, let me fix that up.

Reply:
I looked at acf93a80a9e1b07c9158220c22aef3fcb8a1dd11, now I get it, thanks!